doc/source/dev_guide/custom_reader.rst

```diff
@@ -88,28 +88,37 @@ The ``reader`` section provides basic parameters for the overall reader.
 
 The parameters to provide in this section are:
 
- - name: This is the name of the reader, it should be the same as the
-   filename (without the .yaml extension). The naming convention for
-   this is described above in the :ref:`reader_naming` section above.
- - short_name (optional): Human-readable version of the reader 'name'.
-   If not provided, applications using this can default to taking the 'name',
-   replacing ``_`` with spaces and uppercasing every letter.
- - long_name: Human-readable title for the reader. This may be used as a
-   section title on a website or in GUI applications using Satpy. Default
-   naming scheme is ``<space program> <sensor> Level <level> [<format>]``.
-   For example, for the ``abi_l1b`` reader this is ``"GOES-R ABI Level 1b"``
-   where "GOES-R" is the name of the program and **not** the name of the
-   platform/satellite. This scheme may not work for all readers, but in
-   general should be followed. See existing readers for more examples.
- - description: General description of the reader. This may include any
-   `restructuredtext <http://docutils.sourceforge.net/docs/user/rst/quickref.html>`_
-   formatted text like links to PDFs or sites with more information on the
-   file format. This can be multiline if formatted properly in YAML (see
-   example below).
- - sensors: The list of sensors this reader will support. This must be
-   all lowercase letters for full support throughout in Satpy.
- - reader: The main python reader class to use, in most cases the
-   ``FileYAMLReader`` is a good choice.
+ name
+    This is the name of the reader, it should be the same as the
+    filename (without the .yaml extension). The naming convention for
+    this is described above in the :ref:`reader_naming` section above.
+    short_name (optional): Human-readable version of the reader 'name'.
+    If not provided, applications using this can default to taking the 'name',
+    replacing ``_`` with spaces and uppercasing every letter.
+ long_name
+    Human-readable title for the reader. This may be used as a
+    section title on a website or in GUI applications using Satpy. Default
+    naming scheme is ``<space program> <sensor> Level <level> [<format>]``.
+    For example, for the ``abi_l1b`` reader this is ``"GOES-R ABI Level 1b"``
+    where "GOES-R" is the name of the program and **not** the name of the
+    platform/satellite. This scheme may not work for all readers, but in
+    general should be followed. See existing readers for more examples.
+ description
+    General description of the reader. This may include any
+    `restructuredtext <http://docutils.sourceforge.net/docs/user/rst/quickref.html>`_
+    formatted text like links to PDFs or sites with more information on the
+    file format. This can be multiline if formatted properly in YAML (see
+    example below).
+ status
+    The status of the reader (e.g. Nominal, Beta, etc.)
```

Comment:
Yes I think that would suffice. Will add Alpha.

Reply:
@djhoese so I kept the comments in the status column for now since only a few of the readers hat status comments and it felt overkill to have a extra column which is mostly empty. It looks like this now:

<img src="https://user-images.githubusercontent.com/12115839/169977382-653dd3bd-b011-4611-bdee-bb461d3ca0eb.png" height="600">

I added questionmarks for readers for which the status was unclear for me (see my comment below)